Sources/Target/Linux/Process.cpp

```diff
@@ -37,6 +37,9 @@ using ds2::Utils::Stringify;
 
 #define super ds2::Target::POSIX::ELFProcess
 
+#define STR_HELPER(s) #s
```

Comment:
Argument to macros should be in capital letters, and you should put this in a shared location. `Headers/DebugServer2/Utils/String.h` seems like a good candidate.